source/docs/casper/developers/dapps/signer-integration.md

```diff
@@ -0,0 +1,123 @@
+# Casper Signer in dApp Development
+
+The Casper Signer is a wallet developed by the teams at MAKE and CasperLabs for using Casper accounts. You can interact with the Casper Signer using the `Signer` (Reword this for god sakes).
+
```

Comment:
Reword?

Reply:
Thanks, forgot I had this in here. Thought it could've been re-worded better.